app/src/main/java/org/oppia/app/drawer/NavigationDrawerItem.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.app.drawer
+
+import org.oppia.app.R
+
+enum class NavigationDrawerItem(val value: Int) {
```

Comment:
Please add comment for this class.

Reply:
/** Enum to store the menu of [NavigationDrawerFragment] and get menu by id. */

Reply:
added /** Enum to store the menu of [NavigationDrawerFragment] and get menu by id. */